Fix:

error: implicit declaration of function 'XmGetDestination' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

Define a placeholder XmGetDestination, just like AC_CHECK_LIB does.
--- acinclude.m4.orig	2020-12-17 02:57:21.000000000 -0600
+++ acinclude.m4	2020-12-17 04:34:20.000000000 -0600
@@ -433,7 +433,7 @@
   # Don't add to $LIBS permanently.
   ac_save_LIBS="$LIBS"
   LIBS="-l$xm_direct_test_library $LIBS"
-AC_LINK_IFELSE([AC_LANG_PROGRAM([[]], [[${xm_direct_test_function}()]])],[LIBS="$ac_save_LIBS"
+AC_LINK_IFELSE([AC_LANG_PROGRAM([[char ${xm_direct_test_function} ();]], [[${xm_direct_test_function}()]])],[LIBS="$ac_save_LIBS"
 # We can link Motif programs with no special library path.
 xm_libraries=],[LIBS="$ac_save_LIBS"
 # First see if replacing the include by lib works.
